config/features.yml

```diff
@@ -1,11 +1,13 @@
 ---
-# Add a new feature toggle here to ensure that it is initialized in all environments.
-# Features are defaulted to enabled in development and test environments and disabled in all others.
+# Add a new feture toggle here to ensure that it is initialized in all environments.
+#
+# Features are enabled by default in test environments, and disabled by default in other environments.
+# Set a `enable_in_development` to 'true' to override this behavior for development.
```

Comment:
This line sounds like it was written for the commit rather than this file as it looks like it's already set to `true` for all features. Should the copy read `false` here? 😕